### PR TITLE
Calib containers

### PR DESCRIPTION
**Pull request by FrancaCassol**
- Added first version of the  WaveformCalibrationContainer which contain the waveform calibration coefficients 
- Improved the container description 

This PR must be approved before the last version of the calibration code can be PR

### CODE REVIEW

**Comment by FrancaCassol** [comment]
If nobody is against I would merge this ...

**Comment by vuillaut** [comment]
> If nobody is against I would merge this ...

I can quickly review the code before this evening.

**Comment by FrancaCassol** [comment]
Thanks!

File: ctapipe_io_lst/containers.py [hunk 1]
```diff
@@ -130,28 +133,60 @@ class PixelStatusContainer(Container):
     """
     hardware_mask = Field(
         None,
-        "Mask from the hardware pixel status data (n_pix)"
+        "Boolean np array (mask) from the hardware pixel status data (n_chan X n_pix)"
```
Comment (vuillaut) [inline ctapipe_io_lst/containers.py]: Is it a shape `(n_chan, n_pix)` or a shape `(n_chan*n_pix, )` ?

**Pull request merged**